addons/account/wizard/accrued_orders.py

```diff
@@ -179,8 +179,15 @@ def _ellipsis(string, size):
                 for order_line in lines:
                     if is_purchase:
                         account = self._get_computed_account(order, order_line.product_id, is_purchase)
-                        amount = self.company_id.currency_id.round(order_line.qty_to_invoice * order_line.price_unit / rate)
-                        amount_currency = order_line.currency_id.round(order_line.qty_to_invoice * order_line.price_unit)
+                        # As included taxes are not taken into account in the price_unit, we need to compute the price_subtotal
+                        price_subtotal = order_line.taxes_id.compute_all(
```

Comment:
`compute_all` is a behemoth of a function... Could we maybe use it inside a condition, `if tax_included` style ?

Reply:
@casp-odoo I updated the commit by adding a condition. Tell me if this is what you had in mind.